pyexchange/coinbase.py

```diff
@@ -318,13 +357,15 @@ def _http_unauthenticated(self, method: str, resource: str, body: dict):
                                              data=data,
                                              timeout=self.timeout))
 
-    def _result(self, result) -> Optional[dict]:
+    @staticmethod
+    def _result(result) -> Optional[dict]:
         if not result.ok:
-            raise Exception(f"Coinbase API invalid HTTP response: {http_response_summary(result)}")
-
-        try:
-            data = result.json()
-        except Exception:
-            raise Exception(f"Coinbase API invalid JSON response: {http_response_summary(result)}")
-
-        return data
+            raise RuntimeError(f"Coinbase API response: {http_response_summary(result)}")
+
+        if result.content and result.content != b'OK':
```

Comment:
Hey Ed, could you explain why you're using the byte literal here? Is this the value stored in `result.content`? I assume this is some of the "reworking" you were referring to

Reply:
Yes, that's the success response when Coinbase doesn't return JSON.